bin/aq.py

```diff
@@ -347,7 +347,7 @@ def get_default_opts(auth_option, conf_file=None):
         default_aqhost = defaultOpts.get('aqhost') or 'nyaqd1'
         default_aqservice = 'cdb'
     else:
-        default_aqhost = socket.gethostname()
+        default_aqhost =  defaultOpts.get('aqhost') or socket.gethostname()
```

Comment:
please leave the 'default_aqhost' as it is.. and just add the 'defaultOpts.get('aqhost')' to the line #353  after global options. Same as it is done for service:
host = globalOptions.get('aqhost') or defaultOpts.get('aqhost') or os.environ.get('AQHOST', None) or \
        default_aqhost